circuit-test/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 Slack Technologies, LLC
 // SPDX-License-Identifier: Apache-2.0
+import org.jetbrains.dokka.DokkaDefaults.moduleName
```

Comment:
Was this used?

Reply:
Nope, AS flailing around trying to find something to auto-import, tyty